src/execution_plan/ops/op_cond_var_len_traverse.c

```diff
@@ -96,7 +95,7 @@ OpBase *NewCondVarLenTraverseOp(const ExecutionPlan *plan, Graph *g, AlgebraicEx
 
 	// Populate edge value in record only if it is referenced.
 	AST *ast = QueryCtx_GetAST();
-	QGEdge *e = QueryGraph_GetEdgeByAlias(op->qg, AlgebraicExpression_Edge(op->ae));
+	QGEdge *e = QueryGraph_GetEdgeByAlias(plan->query_graph, AlgebraicExpression_Edge(op->ae));
```

Comment:
Wouldn't you prefer the operation to have query-graph as one of its fields?

Reply:
Not really, because if this op is made for a mock AST, then it would be pointing to an invalid QueryGraph if initialized during `NewCondVarLenTraverse`.

A QueryGraph member could be safely set during the `Init` routine, but I'd just as soon place the responsibility of accessing the QueryGraph on the owner ExecutionPlan.